modules/nodes-nodes-audit-log-basic-config.adoc

```diff
@@ -0,0 +1,175 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-audit-log.adoc
+
+[id="nodes-nodes-audit-log-basic-config-{context}"]
+= Configuring the API Audit Log
+
+
+You can configure the the audit feature to set log level, retention policy, and the type of events to log.
+
+//This is a best guess attempt inspired by https://github.com/openshift/api/pull/279
+
+.Prerequisite
+
+* Cluster logging and Elasticsearch must be installed.
+
+* Set cluster logging to the unmanaged state.
+
+* If needed, get the name of the Cluster Logging Custom Resource in the `openshift-logging` project:
++
+----
+$ oc get ClusterLogging
+NAME       AGE
+instance   112m
+----
+
+.Procedure
+
+. Set the audit log level:
+ 
+.. Edit the Cluster Logging Custom Resource (CR) in the `openshift-logging` project: 
++
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  name: "instance"
+
+....
+
+spec:
+  logLevel: "Normal" <1>
+  operatorloglevel: "TraceAll" <2>
+----
+<1> Specify the log level for an overall component.  
+<2> Specify the log level for the operator itself.
++
+You can set the log level for both settings to one of the following. The setting can be different for each setting:
++
+* *Normal*. Normal is the default.  Normal, working log information, everything is fine, but helpful notices for auditing or common operations. Similar to `glog=2`.
+* *Debug*. Debug is for troubleshooting problems. Even common operations may be logged, and less helpful but more quantity of notices.  Similar to `glog=4`.
+* *Trace*. Trace is for troubleshooting problems when *Debug* is not verbose enough. Logging every function call as part of a common operation, to tracing execution of a query.  Similar to `glog=6`.
+* *TraceAll*. TraceAll is troubleshoting at the level of API content/decoding.  It will dump complete body content. In production clusters, this setting causess performance degradation and results in a significant number of logs.
+Similar to ` glog=8`.
+
+. Configure the audit parameters
+
+.. Edit the auditing configuration file on the API server pod:
++
+----
+$ oc -n openshift-kube-apiserver exec <kube-api-server-name> cat /etc/kubernetes/static-pod-resources/configmaps/config/config.yaml |json_reformat 
+----
+
+.. Edit the `auditConfig` stanza as needed:
```

Comment:
How to edit ?  Seems all by hard code.

Reply:
@deads2k How does the user edit the audit configuration file? Or, can the user not edit? The above command displays the file, but doesn't allow editing.